src/tests/suite/daos_container.c

```diff
@@ -2203,6 +2204,52 @@ co_rf_simple(void **state)
 	test_teardown((void **)&arg);
 }
 
+static void
+delet_container_during_aggregation(void **state)
+{
+	test_arg_t	*arg = *state;
+	daos_obj_id_t	 oid;
+	daos_pool_info_t pinfo;
+	int		i;
+	int		rc;
+
+	/* Prepare records */
+	oid = dts_oid_gen(OC_RP_2G1, 0, arg->myrank);
+
+	print_message("Initial Pool Query\n");
+	pool_storage_info(state, &pinfo);
+
+	/* Aggregation will be Hold */
+	daos_debug_set_params(arg->group, -1, DMG_KEY_FAIL_LOC,
+			      DAOS_CONT_AGG_YIED | DAOS_FAIL_ALWAYS, 0, NULL);
+
+	/* Write/fetch and Punch Data with 2K size */
+	for (i = 0; i <= 50; i++)
```

Comment:
Maybe larger dataset to make aggregation last longer?

Reply:
I tested and this worked so though not to write more data but can increase

Reply:
The test is like: 1. block aggregation; 2. unblock aggregation; 3. destroy container; so I hope the aggregation not finishing before the 3rd step. Maybe larger dataset could be helpful.